packages/page-statistics/src/containers/KeygenTable/KeygenTable.tsx

```diff
@@ -218,6 +218,8 @@ export const KeygenTable: FC = () => {
         info: 'List of Keygens',
         variant: 'h5',
       }}
+      /* Temporarily disable filter functionality and will support it in the near future
```

Comment:
Say more here? Is there a reason filtering would be disabled?

Reply:
It's not working right now, it throws an error when you try to change the slider. Actually, I will open an issue for it and enable this functionality.